### PR TITLE
Stef's draft unconf17 recap blog post

### PR DESCRIPTION
**Pull request by stefaniebutland**

### CODE REVIEW

**Comment by stefaniebutland** [comment]
@karthik and @sckott I need some feedback and input on this post. Would like to have it up Thurs June 1st if possible.

- can one of you give nice text for 3 bullet point examples of unconf projects to highlight - see placeholder in post
- appreciate any feedback on text
- I like the whiteboard photo next to a working group photo, but the whiteboard photo looks poor resolution - solution?
- I don't have any pics of working groups that look really dynamic
- Karthik do you have pics you really want to feature? I chose not to highlight rOpenSci team.
- I know whiteboard and working group photos are diff sizes but didn't want to perfect if we were going to get rid of one.
- can't figure out how to add appropriate space around R stickers pic or around the embedded tweets
- How should I list photo credits?

Thanks!

**Comment by sckott** [comment]
@stefaniebutland will look

before 9 am or after?

**Comment by stefaniebutland** [comment]
@sckott after 9

**Comment by sckott** [comment]
> can one of you give nice text for 3 bullet point examples of unconf projects to highlight - see placeholder in post

I guess need to decide which ones first. these seem interesting to highlight:

* https://github.com/ropenscilabs/notary & https://github.com/ropenscilabs/r-security-practices (as one)
( 
* https://github.com/ropenscilabs/skimr - very popular already
* https://github.com/ropenscilabs/emldown
* https://github.com/ropenscilabs/packagemetrics

> appreciate any feedback on text

will make comments in line

> I like the whiteboard photo next to a working group photo, but the whiteboard photo looks poor resolution - solution?

looks okay to me

> I know whiteboard and working group photos are diff sizes but didn't want to perfect if we were going to get rid of one.

can just play with width of left one, e.g., 39% looks pretty good, or maybe 40 would be better. close enough i think

> can't figure out how to add appropriate space around R stickers pic or around the embedded tweets

don't know, maybe karthik knows

File: _drafts/2017-06-01-unconf2017.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- ropensci
+- meetings
+- unconf
+
+---
+
+70 people from 7 countries on 3 continents, more than 70 ideas leading to 21 projects in 2 days, and one awesome community that just upped its game! We held our [4th annual unconference](http://unconf17.ropensci.org/) in Los Angeles, May 25-26, 2017. We brought together scientists, R-software users and developers, and open data enthusiasts from academia, industry, government, and non-profits to get together for a couple of days to hack on projects they dreamed up and to give our online community an opportunity to develop authentic face-to-face relationships.
+
+## Look what we made!
+People proposed and discussed [project ideas in GitHub issues](https://github.com/ropensci/unconf17/issues/) in the weeks leading up to the unconf to the point that we had more projects than participants. Participants sticker-voted for projects they’d like to work on in order to reduce the number under consideration, and after some brief discussions and self-sorting, teams were working away around the [Cross Campus DTLA](http://www.crosscamp.us/site/locations/cross-campus-downtown-la.html) co-working space for the next day and a half.
```
Comment (sckott) [inline _drafts/2017-06-01-unconf2017.md]: maybe `more project ideas than participants`

File: _drafts/2017-06-01-unconf2017.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Join the lunchtime <a href="https://twitter.com/hashtag/runconf17?src=hash">#runconf17</a> discussion about the <a href="https://twitter.com/hashtag/rstats?src=hash">#rstats</a> communities - what do we need to do to improve? <a href="https://t.co/ztbXxNfqU7">pic.twitter.com/ztbXxNfqU7</a></p>&mdash; Alice Data (@alice_data) <a href="https://twitter.com/alice_data/status/868198529134510080">May 26, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Speaking of awesome community, [R-Ladies](http://rladies.org/), a world-wide organization to promote gender diversity in the R community, was incredibly well-represented with many founders of local chapters present. ![ropensci-unconf17-rladies](/assets/blog-images/2017-06-01-unconf17/ropensci-unconf17-rladies.jpg)
```
Comment (sckott) [inline _drafts/2017-06-01-unconf2017.md]: https version https://rladies.org/

File: _posts/2017-06-01-unconf2017.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+<!---<img src="/assets/blog-images/2017-06-01-unconf17/ropensci-unconf17-whiteboard1.jpg" width="45%" align="left">--->
+
+<img src="/assets/blog-images/2017-06-01-unconf17/ropensci-unconf17-whiteboard2_960_720.jpg" width="45%"  align="left">
+<img src="/assets/blog-images/2017-06-01-unconf17/ropensci-unconf17-workgroups_1620_1080.jpg" width="45%">
+
+Not only did our funders and sponsors - Helmsley Charitable Trust, [Alfred P. Sloan Foundation](http://sloan.org/), [Microsoft](https://www.microsoft.com/en-us/cloud-platform/r-server), [Google](http://www.google.com/), [DataCamp](http://datacamp.com/), and [NumFOCUS](https://www.numfocus.org/) - help us make this happen, but people from Microsoft, Google and DataCamp fully participated in unconf projects!
```
Comment (sckott) [inline _posts/2017-06-01-unconf2017.md]: https://sloan.org/
https://www.google.com/
https://www.datacamp.com/

File: _posts/2017-06-01-unconf2017.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+
+<blockquote class="twitter-tweet" data-cards="hidden" data-lang="en"><p lang="en" dir="ltr">Join the lunchtime <a href="https://twitter.com/hashtag/runconf17?src=hash">#runconf17</a> discussion about the <a href="https://twitter.com/hashtag/rstats?src=hash">#rstats</a> communities - what do we need to do to improve? <a href="https://t.co/ztbXxNfqU7">pic.twitter.com/ztbXxNfqU7</a></p>&mdash; Alice Data (@alice_data) <a href="https://twitter.com/alice_data/status/868198529134510080">May 26, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Speaking of awesome community, [R-Ladies](http://rladies.org/), a world-wide organization to promote gender diversity in the R community, was incredibly well-represented by both founders of the global organization as well as of local chapters.
```
Comment (sckott) [inline _posts/2017-06-01-unconf2017.md]: https version https://rladies.org/

File: _posts/2017-06-01-unconf2017.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+## Look What We Made!
+
+<img src="/assets/blog-images/2017-06-01-unconf17/ropensci-unconf17-stickers.jpg" width="15%" align="left">
+
+People proposed and discussed [project ideas in GitHub issues](https://github.com/ropensci/unconf17/issues/) in the weeks leading up to the unconf to the point that we had more projects than participants. On the morning of Day 1, people sticker-voted for projects they’d like to work on in order to reduce the number under consideration. After some brief discussions and self-sorting, teams settled in to work around the [Cross Campus DTLA](http://www.crosscamp.us/site/locations/cross-campus-downtown-la.html) co-working space for the next day and a half.
```
Comment (sckott) [inline _posts/2017-06-01-unconf2017.md]: maybe `more project ideas than participants`

**Comment by sckott** [comment]
@stefaniebutland 

- Possibly link out to all blog posts about unconf so far, think there's been 4 or so
- and maybe call for more blog posts from unconf participants ?

**Comment by stefaniebutland** [comment]
Do we want to cite 45% women at unconf? and the fact that we pay up front for anyone who needs it so there is no barrier to participation and therefore opens door for diversity?

I could *really really* use suggestion on how to word this

File: _posts/2017-06-01-unconf2017.md (hunk 5 above)
Comment (stefaniebutland) [inline _posts/2017-06-01-unconf2017.md]: noted in previous para so not repeated here

File: _posts/2017-06-01-unconf2017.md (hunk 3 above)
Comment (stefaniebutland) [inline _posts/2017-06-01-unconf2017.md]: done

File: _posts/2017-06-01-unconf2017.md (hunk 4 above)
Comment (stefaniebutland) [inline _posts/2017-06-01-unconf2017.md]: done

File: _posts/2017-06-01-unconf2017.md (hunk 5 above)
Comment (stefaniebutland) [inline _posts/2017-06-01-unconf2017.md]: done

File: _posts/2017-06-01-unconf2017.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+categories: blog
+tags:
+- community
+- ropensci
+- meetings
+- unconf
+
+---
+
+70 people from 7 countries on 3 continents, more than 70 ideas leading to 21 projects in 2 days, and one awesome community that just upped its game! We held our [4th annual unconference](http://unconf17.ropensci.org/) in Los Angeles, May 25-26, 2017. We brought together scientists, R-software users and developers, and open data enthusiasts from academia, industry, government, and non-profits to get together for a couple of days to hack on projects they dreamed up and to give our online community an opportunity to develop authentic face-to-face relationships.
```
Comment (stefaniebutland) [inline _posts/2017-06-01-unconf2017.md]: incorporated suggestions

File: _drafts/2017-06-01-unconf2017.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+Not only did our funders and sponsors - Helmsley Charitable Trust, Alfred P. Sloan Foundation, Microsoft, Google, DataCamp, NumFOCUS - help us make this happen, but people from Microsoft, Google and DataCamp fully participated in unconf projects!
+
+Decide whether and where to put these items:
+- we debuted new rOpenSci stickers! ![pic-name](/assets/blog-images/2017-05-31-unconf17/pic_name.jpg)
+- Our Project Lead, Karthik Ram, [storified unconf17 tweets](https://storify.com/_inundata/4th-annual-ropensci-unconf)
+- Soak up the enthusiasm for what happened via twitter hashtag [#runconf17](https://twitter.com/search/?q=%23runconf17)
+
+## The community is one of #rstats best assets
```
Comment (stefaniebutland) [inline _drafts/2017-06-01-unconf2017.md]: done

File: _drafts/2017-06-01-unconf2017.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+<blockquote class="twitter-tweet" data-partner="tweetdeck"><p lang="en" dir="ltr">For a FULL LIST of <a href="https://twitter.com/hashtag/runconf17?src=hash">#runconf17</a> <a href="https://twitter.com/hashtag/rstats?src=hash">#rstats</a> projects here is a link (including awesome emojis) 📦🎉⭐️💻<a href="https://t.co/FsoOu5gUAr">https://t.co/FsoOu5gUAr</a> <a href="https://t.co/gHbZOfBNTV">pic.twitter.com/gHbZOfBNTV</a></p>&mdash; Alice Data (@alice_data) <a href="https://twitter.com/alice_data/status/868336229992890368">May 27, 2017</a></blockquote>
+
+Not only did our funders and sponsors - Helmsley Charitable Trust, Alfred P. Sloan Foundation, Microsoft, Google, DataCamp, NumFOCUS - help us make this happen, but people from Microsoft, Google and DataCamp fully participated in unconf projects!
+
+Decide whether and where to put these items:
+- we debuted new rOpenSci stickers! ![pic-name](/assets/blog-images/2017-05-31-unconf17/pic_name.jpg)
```
Comment (stefaniebutland) [inline _drafts/2017-06-01-unconf2017.md]: removed text about stickers and just kept pic

**Comment by stefaniebutland** [comment]
@karthik could you please give me ~ 1 bullet point sentence each for just 3 of the following projects? I will add to this just before the paragraph about reporting out

- https://github.com/ropenscilabs/notary & https://github.com/ropenscilabs/r-security-practices (as one)
(
- https://github.com/ropenscilabs/skimr - very popular already
- https://github.com/ropenscilabs/emldown
- https://github.com/ropenscilabs/packagemetrics
- research compendium

**Comment by stefaniebutland** [comment]
@sckott I think it's ready to post!
Can you change the filename to match June 2? Would be ideal to get it up before 9am Friday but I'm kinda late so no worries if not.

Thank you @karthik and @sckott for all your help - it's much better now I think

**Pull request merged**